app/controllers/items/history.js

```diff
@@ -0,0 +1,74 @@
+import detail from "./detail";
+import Ember from "ember";
+import _ from "lodash";
+
+export default detail.extend({
+  enableFooterActions: false,
+
+  groupedVersions() {
+    let versions = this.get("model.versions").toArray();
+
+    let results = _.map(versions, function(version) {
+      return _.omit(version.get("objectChanges"), [
+        "available_quantity",
+        "designated_quantity",
+        "on_hand_quantity",
+        "dispatched_quanitity",
+        "id",
+        "detail_type_id",
+        "created_at",
+        "received_at"
+      ]);
+    });
+    versions.forEach((version, index) =>
+      version.set("objectChanges", results[index])
+    );
```

Comment:
Don't modify data, recreate a fresh set and return that instead (pure functions)